ivy/functional/ivy/manipulation.py

```diff
@@ -541,15 +541,103 @@ def squeeze(
         an output array having the same data type and elements as ``x``.
 
 
-    Examples
-    --------
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+
     >>> x = ivy.array([[[0, 1], [2, 3]]])
     >>> print(x.shape)
     (1, 2, 2)
 
     >>> print(ivy.squeeze(x, axis=0).shape)
     (2, 2)
 
+    >>> print(ivy.squeeze(x).shape)
+    (2, 2)
+
+    >>> x = ivy.array([[[[1, 2, 3]], [[4, 5, 6]]]])
+    >>> print(x.shape)
+    (1, 2, 1, 3)
+
+    >>> print(ivy.squeeze(x, axis=2).shape)
+    (1, 2, 3)
+
+    >>> x = ivy.array([[[0], [1], [2]]])
+    >>> print(x.shape)
+    (1, 3, 1)
+
+    >>> print(ivy.squeeze(x))
+    [0, 1, 2]
+
+    >>> print(ivy.squeeze(x, axis=0))
+    [[0], [1], [2]]
+
+    >>> print(ivy.squeeze(x, axis=2))
+    [[0, 1, 2]]
+
+    >>> print(ivy.squeeze(x, axis=(0, 2)))
+    [0, 1, 2]
+
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([0, 1, 2])
+    >>> print(ivy.squeeze(x))
+    ivy.array([0, 1, 2])
+
+    >>> x = ivy.native_array([[[3]]])
+    >>> print(x.shape)
+    (1, 1, 1)
+
+    >>> print(ivy.squeeze(x, 2))
+    ivy.array([[3]])
+
+    >>> x = ivy.native_array(0)
+    >>> print(x.shape)
+    ()
+
+    >>> print(ivy.squeeze(x, 0))
+    ivy.array(0)
+
+    With :code:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([0., 1., 2.]), \
+                          b=ivy.array([3., 4., 5.]))
+    >>> y = ivy.squeeze(x)
+    >>> print(y)
+    {
+        a: ivy.array([0., 1., 2.]),
+        b: ivy.array([3., 4., 5.])
+    }
+
+    Instance Method Examples
```

Comment:
Hi, sorry to be so nitpicky, could you just remove the instance method examples from here, since you've already added them to container and arrays, these are redundant. Thanks.